azure-pipelines/build.yml

```diff
@@ -210,10 +210,4 @@ steps:
     command: push
     packagesToPush: '$(Build.SourcesDirectory)/artifacts/nugetPackages/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
     publishVstsFeed: '2191dd5f-4aec-491b-ac50-568bbc331c8a'
-    allowPackageConflicts: true
-
-- task: PublishTestResults@2
```

Comment:
Why are you removing this?

Reply:
I'm looking through this doc on Azure Pipelines and it seems like it's not necessary because it's being done by the Microsoft.NET.Test.SDK reference in the test projects. I linked the doc below. Atm I'm just pushing stuff and testing builds to ensure everything still works.

https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/dotnet-core?view=azure-devops&tabs=dotnetfive#collect-code-coverage

Reply:
Atm I'm running into an issue with the code coverage for the Kusto.ServiceLayer and Sql.ServiceLayer being combined into a single file

Reply:
I see. Could you convert this into a draft PR until it's ready to review then?

Reply:
The reason I started looking into this further is because I was seeing this warning during the builds

![image](https://user-images.githubusercontent.com/63619224/135688129-999ef9af-a683-49cd-8b48-e9f304f2635c.png)

Here's a link to that build.
https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=122194&view=results